src/main/java/model/Lotto.java

```diff
@@ -0,0 +1,54 @@
+package model;
+
+import java.util.*;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class Lotto {
+
+    // 로또 번호 관련 상수 선언
+    public static final int LOTTO_MIN_NUMBER = 1;
+    public static final int LOTTO_MAX_NUMBER = 45;
+    public static final int LOTTO_CREATE_SIZE = 6;
+    public static final int LOTTO_PRICE = 1000;
+    public static final List<Integer> LOTTO_NUMBER_POOL =
+            IntStream
+                    .rangeClosed(LOTTO_MIN_NUMBER,LOTTO_MAX_NUMBER)
+                    .boxed()
+                    .collect(Collectors.toList());
+
+    private List<Integer> numbers = new ArrayList<>();
+
+    public static int getTicketCount(int purchaseAmount){
+        return purchaseAmount / LOTTO_PRICE;
+    }
```

Comment:
```suggestion
    public static int getTicketCount(int purchaseAmount) {
        return purchaseAmount / LOTTO_PRICE;
    }
```

중괄호 앞은 관례상 공백을 추가하는 것이 좋습니다. 수정해주시면 좋을 것 같아요!

포맷팅 단축키를 이용해보세요 😉

Reply:
추가로 1000원 단위가 아닌 값을 입력했을 때는 어떻게 될까요? 🧐

Reply:
현재는 1000원단위로 끊어지므로 1300을 입력하면 1회 시행하게 됩니다.
로또를 1.5회 사거나 하는 경우는 없으니
1000원 단위의 입력만 인식할 수 있도록 예외처리해주는 것이 좋겠습니다.